cernopendata_client/config.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# cernopendata-client is free software; you can redistribute it and/or modify
+# it under the terms of the GPLv3 license; see LICENSE file for more details.
+
+"""cernopendata-client configuration."""
+
+import os
+
+"""Default CERN Open Data query server"""
```

Comment:
Two things:

- The docstring comes after the variable, not before it.
- Please amend the text to "Default CERN Open Data portal instance to query."

Reply:
Done 👍